.travis.yml

```diff
@@ -13,6 +13,7 @@ cache:
   pip:
   directories:
     - dynamodb
+    - minio
```

Comment:
i'm not so keen on testing with minio, you can mock s3 with botocore.stubber https://botocore.readthedocs.io/en/latest/reference/stubber.html